coursedata/course/online_masters/nl.yaml

```diff
@@ -1,150 +1,12 @@
 ---
 course:
-  -
-    level: "1"
-    assignments:
-      - step: "1"
-        prompt: |
-          <a href="https://youtu.be/LKnQvUvCRqM" target="_blank">Klik hier om de uitlegvideo te bekijken.</a>
```

Comment:
Are you okay losing the YouTube video links? I don't think those are in the regular adventures, are they?

Reply:
Yeah we can live without them (they are really old by now anyway)